src/FakeItEasy/Generated/CodeGen/CodeGen.AsyncReturnValueConfigurationExtensionsGenerator/AsyncReturnValueConfigurationExtensions.StronglyTyped.cs

```diff
@@ -22,7 +20,6 @@ public static partial class AsyncReturnValueConfigurationExtensions
         /// <param name="exceptionFactory">A function that returns the exception to set on the returned task when a call that matches is invoked.</param>
         /// <typeparam name="T1">The type of the first argument of the faked method call.</typeparam>
         /// <returns>The configuration object.</returns>
-        [SuppressMessage("Microsoft.Design", "CA1006:DoNotNestGenericTypesInMemberSignatures", Justification = "This is by design to support the fluent API.")]

```

Comment:
Turns out we weren't triggering a warning anymore, so a bonus change. Applies elsewhere.